### PR TITLE
Very rough initial policy detail page

### PR DESCRIPTION
**Pull request by cmc333333**
It lists requirements! They can be clicked!

This gives us a structure to build off more than a full implementation.

Looks like
policy-detail

### CODE REVIEW

**Comment by cmc333333** [comment]
Thanks @rememberlenny ! @jeremiak does anything stick out wrt the direction we're trying to take around code layout?

**Comment by rememberlenny** [comment]
Ill leave this up to @jeremiak

File: ui/components/policy/page-counter.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+
+export default function PageCounter({ count, page, pageSize }) {
```
Comment (jeremiak) [inline ui/components/policy/page-counter.js]: This component looks good and tested, but it doesn't look like its policy specific so it should be in `components/`. I think the benefit of this is that it is easy to find all the components in one spot, leading to increased chance of reuse and refactor.

File: ui/pages/policy.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+export class Policy extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      focusReq: props.url.query.reqId,
+    };
+  }
+
+  get reqs() {
```
Comment (jeremiak) [inline ui/pages/policy.js]: I don't think we use the class getter/setter methods anywhere else in the code base so for consistency I would change this to just be `reqs`.

FWIW, I think using the getter/setters might add some readability to the code base but it should be a consistent pattern across the component if not also the codebase

**Comment by jeremiak** [comment]
@cmc333333 ya, I think the general direction is fine. One thing I'd suggest is to put as many components in a single directory as you feel comfortable doing. At some point, this list gets unwieldy or requires long file, unique file names, but up until that point it makes it easier to know which components are available for re-use/refactoring

**Comment by cmc333333** [comment]
Thanks @jeremiak , will do.

**Pull request merged**